src/main.js

```diff
@@ -30,72 +30,150 @@ app.whenReady().then(() => {
-                    }
+                    },
                 },
-                { role: "quit" }
-            ]
+                isMac ? { role: "close" } : { role: "quit" },
+            ],
         },
+        // { role: 'editMenu' }
         {
-            label: "&View",
+            label: "&Edit",
+            submenu: [
+                { role: "undo" },
+                { role: "redo" },
+                { type: "separator" },
+                { role: "cut" },
+                { role: "copy" },
+                { role: "paste" },
+                ...(isMac
+                    ? [
+                        { role: "pasteAndMatchStyle" },
+                        { role: "delete" },
+                        { role: "selectAll" },
+                        { type: "separator" },
+                        {
+                            label: "Speech",
+                            submenu: [
+                                { role: "startSpeaking" },
+                                { role: "stopSpeaking" }
+                            ],
+                        },
+                    ]
+                    : [
+                        { role: "delete" },
+                        { type: "separator" },
+                        { role: "selectAll" }
+                    ]),
+            ],
+        },
+        // { role: 'viewMenu' }
+        {
+            label: "View",
             submenu: [
                 {
                     label: "Matches",
                     accelerator: "CmdOrCtrl+M",
                     click: (_, browserWindow) => {
-                        browserWindow.webContents
-                            .send("show-project-pairs-view");
-                    }
+                        browserWindow.webContents.send(
+                            "show-project-pairs-view"
+                        );
+                    },
                 },
                 {
                     label: "Warnings",
                     accelerator: "CmdOrCtrl+W",
                     click: (_, browserWindow) => {
                         browserWindow.webContents.send("show-warnings-view");
-                    }
+                    },
                 },
                 { type: "separator" },
-                { role: "zoomIn", accelerator: "CmdOrCtrl+=" },
-                { role: "zoomOut" },
+                { role: "reload" },
+                { role: "forceReload" },
+                { role: "toggleDevTools" },
+                { type: "separator" },
                 { role: "resetZoom" },
-                { role: "togglefullscreen" },
+                { role: "zoomIn" },
+                { role: "zoomOut" },
                 { type: "separator" },
-                { role: "toggleDevTools" },
-            ]
+                { role: "togglefullscreen" },
+            ],
+        },
+        // { role: 'windowMenu' }
+        {
+            label: "Window",
+            submenu: [
+                { role: "minimize" },
```

Comment:
The default keyboard shortcuts for minimize and close (CTRL+M and CTRL+W) clash with the ones I chose for opening the matches and warning views. Maybe we should change the latter to CTRL+SHIFT+M and CTRL+SHIFT+W or something.

Reply:
Sure! Fixed it